Plugins/GitUIPluginInterfaces/GitRevision.cs

```diff
@@ -98,7 +99,11 @@ private static DateTime FromUnixTimeSeconds(long unixTime)
         }
 
         public bool HasMultiLineMessage { get; set; }
-        public bool HasNotes { get; set; }
+
+        /// <summary>
+        /// Git Notes https://git-scm.com/docs/git-notes. Null means "not loaded yet", string.Empty means "empty".
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Git notes (see https://git-scm.com/docs/git-notes).
        /// </summary>
        /// <remarks>
        /// <see langword="null"/> means "notes not loaded yet", <see cref="string.Empty"/> means "no notes".
        /// </remarks>
```

Reply:
Even better with "see href" for git-scm.